en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+    6. Now let’s extract the total case count from the response and store it in a variable. Follow this procedure:
+
+        1. Click the last **+** icon in the low-code diagram.
+
+        2. Under **Statements**, select **Variable** and enter details as follows:
+
+            | **Field**      | **Value**                     |
+            |----------------|-------------------------------|
+            | **Type**       | `int`                         |
+            | **Name**       | `totalCases`                  |
+            | **Expression** | `<int>statusByCountry.cases`  |
+
+        3. Click **Save**.
+
+5. To retrieve the population data, connect to the World Bank API as follows:
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Click **Connector** and then select **World Bank API**.
+
+    3. In the **World Bank API Connection** window, enter `worldBankClient` as the **Connection Name** and click **Continue to Invoke API**.
+
+    4. In the **Operation** drop-down list, select **getPopulationByCountry** and enter details as follows in the other fields:
+
+        | **Field**                  | **Value**            |
+        |----------------------------|----------------------|
+        | **Country Code**           | `country`            |
+        | **Response Variable Name** | `populationByCountry`|
+
+    5. Click **Save**.
+
+    6. Now let’s extract the population value from the response, calculate the population in millions, and store it in a variable. Follow this procedure:
+
+        1. Click the last **+** icon in the low-code diagram.
+
+        2. Under **Statements**, select **Variable** and enter details as follows:
+
+            | **Field**      | **Value**                                         |
+            |----------------|---------------------------------------------------|
+            | **Type**       | `int`                                             |
+            | **Name**       | `populationMillions`                              |
+            | **Expression** | `(populationByCountry[0]?.value ?: 0) / 1000000`  |
+
+        3. Click **Save**.
+
+6. Now let’s calculate the total COVID-19 case count per million in the population based on the COVID-19 statistics and the population data you have retrieved. Follow this procedure:
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Under **Statements**, select **Variable** and enter details as follows:
+
+        | **Field**      | **Value**                                    |
+        |----------------|----------------------------------------------|
+        | **Type**       | `decimal`                                    |
+        | **Name**       | `totalCasesPerMillion`                       |
+        | **Expression** | `<decimal>(totalCases / populationMillions)` |
+
+    3. Click **Save**.
+
+7. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
```

Comment:
Shall we mention why we are building a JSON payload?

Reply:
Should JSON be in ``. in the step below it is.

Reply:
Removed the code format in the step below.